angular-workspace/.gitignore

```diff
@@ -31,6 +31,7 @@ chrome-profiler-events*.json
 .history/*
 
 # misc
```

Comment:
Looks like some old scss tilde imports are left in docs and generated files:

- https://github.com/ni/nimble/blob/942893b95ff2d24c8e6bd4dd71169cde2b185191/packages/nimble-components/README.md?plain=1#L78
- https://github.com/ni/nimble/blob/942893b95ff2d24c8e6bd4dd71169cde2b185191/packages/nimble-components/README.md?plain=1#L86
- https://github.com/ni/nimble/blob/942893b95ff2d24c8e6bd4dd71169cde2b185191/packages/nimble-components/build/generate-scss/source/index.js#L16-L17
- https://github.com/ni/nimble/blob/942893b95ff2d24c8e6bd4dd71169cde2b185191/packages/nimble-tokens/README.md?plain=1#L43

Reply:
@TrevorKarjanis looks like one of the above linked references wasn't updated:

https://github.com/ni/nimble/blob/8b0be49b3e91e145dfcf834ebf89d6fb5ffbcf5d/packages/nimble-components/build/generate-scss/source/index.js#L16-L17

Reply:
Resolved.